WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -895,11 +909,11 @@ extension ReaderDetailViewController: UIScrollViewDelegate {
         // Using `safeAreaLayoutGuide.layoutFrame.height` because it doesn't
         // change when we extend the scroll view size by hiding the toolbar
         if (currentOffset + view.safeAreaLayoutGuide.layoutFrame.height) > likesContainerView.frame.minY {
-            setToolbarHidden(false, animated: true) // Reached bottom (controls, comments, etc)
+            toolbarHiddenDebouncer.send(false) // Reached bottom (controls, comments, etc)
```

Comment:
There is a `setNeedsToolbarHidden` method below designed to do the same thing, but I didn't update the call site.